api/src/middleware/InjectionKeys.ts

```diff
@@ -0,0 +1,23 @@
+import { InjectionKey, InjectionKeyScope } from "@corecodeio/libraries/di";
+import { PhoneNumberExistMiddleware } from "./phoneNumberExist";
+import { PhoneNumberVerificationMiddleware } from "./phoneNumberVerification";
+
+export const PhoneNumberVerificationMiddlewareInjectionKey: InjectionKey<PhoneNumberVerificationMiddleware> = {
+  name: "PhoneNumberVerificationMiddlewareInjectionKey",
+  scope: InjectionKeyScope.singleton,
+  closure: (dependencies) => {
+    const onboardingController = new PhoneNumberVerificationMiddleware();
+
+    return onboardingController;
+  },
+};
+
+export const PhoneNumberExistMiddlewareInjectionKey: InjectionKey<PhoneNumberExistMiddleware> = {
```

Comment:
Cuando el sustantivo es singular: `PhoneNumber`, el verbo lleva `s`: phone number exists. 

```suggestion
export const PhoneNumberExistsMiddlewareInjectionKey: InjectionKey<PhoneNumberExistMiddleware> = {
```

Reply:
ya lo modifique en todos los lados que fueron necesarios